### ISSUE TITLE
Only showing memory

### ISSUE THREAD

**Issue by nicdelillo-fitprime** [jupyter-server/jupyter-resource-usage#234]
Hi everyone, I installed the new resource usage over the old system monitor. 
It seems to kinda work but I am not sure why it is showing only the memory and without the upper limit.
Jupyter lab version in 4.1.8

 In my old Jupyter lab 2.1.3 system monitor was working brilliantly and showing CPU percentage and memory used over the slashed max limit in this format"used/max".

Is there anything I can do? I also attache two screenshot of the current status.
<img width="426" alt="Screenshot 2024-05-03 at 22 23 29" src="https://github.com/jupyter-server/jupyter-resource-usage/assets/167303955/247982
<img width="140" alt="Screenshot 2024-05-03 at 22 23 36" src="https://github.com/jupyter-server/jupyter-resource-usage/assets/167303955/8aeedf04-d319-4ac8-b5bf-9b4c2b7d2209">
e6-3886-4d80-8683-564300708429">

**Comment by nicdelillo-fitprime** [comment]
<img width="426" alt="Screenshot 2024-05-03 at 22 23 29" src="https://github.com/jupyter-server/jupyter-resource-usage/assets/167303955/e751683e-ef3d-4dfc-85f1-6ea93a68e203">

**Comment by guzman-raphael** [comment]
@jtpio Ran into this issue today as well. Saw that [jtpio/jupyterlab-system-monitor](https://github.com/jtpio/jupyterlab-system-monitor) has been archived and jupyter-server/jupyter-resource-usage#84 is still open. Is there any solution to get CPU info to display on Jupyter Lab 4.X UI? Memory upper bound also don't show up for me. Do you have to set `MEM_LIMIT` as opposed to detecting available?

**Comment by nicdelillo-fitprime** [comment]
Yes I confirm also for me memory memory upper bound do not show as well

**Comment by TopCoder2K** [comment]
Same here with `jupyterlab=4.2.1` and `jupyter-server-resource-usage=1.0.2`. Are there any plans to fix this, @jtpio?

**Comment by frittentheke** [comment]
Is this related to the PR https://github.com/jupyter-server/jupyter-resource-usage/pull/153 by any chance?

**Comment by Vector-Cross** [comment]
if you change user name or workdir just like below, you should modify `/home/zylj/.jupyter/jupyter_server_config.py` instead of `~/.jupyter/jupyter_server_config.py`
```
docker run -d \
    -p 52202:8888 \
    --user root \
    -e NB_USER="zylj" \
    -e CHOWN_HOME=yes \
    -e GRANT_SUDO=yes \
    -w "/home/zylj" \
    -v "/app/jupyter-sci-lab/workspace":/home/zylj \
    -e TZ="Asia/Shanghai" \
    -v /etc/localtime:/etc/localtime:ro \
    --name jupyter-sci-lab \
    jupyter/scipy-notebook:aarch64-lab-4.0.7
```

**Comment by krassowski** [comment]
@Vector-Cross did you men to post it in this thread? It looks unrelated at first glance.

**Comment by krassowski** [comment]
To show CPU usage you need to follow instructions in the README:

https://github.com/jupyter-server/jupyter-resource-usage#cpu-usage

If you want to show disk usage, again please follow instructions in the README: 

https://github.com/jupyter-server/jupyter-resource-usage#disk-partition-usage

(after making sure you have the latest version installed).
